samples/drivers/display/boards/stm32_min_dev_black.conf

```diff
@@ -0,0 +1,2 @@
+CONFIG_HEAP_MEM_POOL_SIZE=1024
```

Comment:
NACK for board specific overlays in this case. Please take a look at the shield support, you can add a generic MAX7219 shield like  https://github.com/zephyrproject-rtos/zephyr/tree/main/boards/shields/ssd1306

Reply:
OK. I will add MAX7219 into `boards/shields`.
Thanks.